doc/examples/plot_hyperparameter.py

```diff
@@ -130,6 +150,14 @@ def evaluate_hyperparameters(config):
     train(model, optimizer, train_loader)
     return test(model, test_loader)
 
+#######################################################################
+# Synchronous Evaluation of Randomly Generated Hyperparameters
+# ------------------------------------------------------------
+#
+# We will create multiple sets of random hyperparameters for our neural
+# network that will be trained in parallel. These trained networks will
```

Comment:
```suggestion
# network that will be evaluated in parallel. These trained networks will
```

Reply:
Also, I think the last sentence "tested for accuracy..." should be moved to the previous block